docs/cli.md

```diff
@@ -0,0 +1,38 @@
+# Promscale command line doc
+
+This document gives you information about the configuration flags supported by promscale.
+You can also find information on flags with `promscale_<version> -help` 
+
+## General flags
+| Flag | Type | Default | Description |
+|------|:-----:|:-------:|:-----------:|
+| install-timescaledb | boolean | true | Install or update TimescaleDB extension. |
+| leader-election-pg-advisory-lock-id | integer | 0 (disabled) | Leader-election based high-availability. It is based on PostgreSQL advisory lock and requires a unique advisory lock ID per high-availability group. Only a single connector in each high-availability group will write data at one time. A value of 0 disables leader election. |
+| leader-election-pg-advisory-lock-prometheus-timeout | slack/duration | -1 | Prometheus timeout duration for leader-election high-availability. The connector will resign if the associated Prometheus instance does not respond within the given timeout. This value should be a low multiple of the Prometheus scrape interval, big enough to prevent random flips. |
+| log-format | string | logfmt | Log format to use from [ "logfmt", "json" ]. |
+| log-level | string | debug | Log level to use from [ "error", "warn", "info", "debug" ]. |
+| labels-cache-size | unsigned-integer | 10000 | Maximum number of labels to cache. |
+| metrics-cache-size | unsigned-integer | 10000 | Maximum number of metric names to cache. |
+| migrate | string | true | Update the Prometheus SQL schema to the latest version. Valid options are: [true, false, only]. |
+| read-only | boolean | false | Read-only mode for the connector. Operations related to writing or updating the database are disallowed. It is used when pointing the connector to a TimescaleDB read replica. |
+| scheduled-election-interval | duration | 5 seconds | Interval at which scheduled election runs. This is used to select a leader and confirm that we still holding the advisory lock. |
```

Comment:
should this be renamed to `leader-election-scheduled-interval`?

Reply:
Yes, sounds more relateable.